consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/CommitPayload.java

```diff
@@ -55,7 +56,8 @@ public static CommitPayload readFrom(final RLPInput rlpInput) {
   @Override
   public void writeTo(final RLPOutput rlpOutput) {
     rlpOutput.startList();
-    roundIdentifier.writeTo(rlpOutput);
+    rlpOutput.writeLongScalar(roundIdentifier.getSequenceNumber());
+    rlpOutput.writeIntScalar(roundIdentifier.getRoundNumber());
```

Comment:
Bit nasty duplicating this for every message and just ignoring the encode on the ConsensusRoundIdentifier. Perhaps move these into a writeTo or encode util for the qbft round identifier?